ISTC.js

```diff
@@ -0,0 +1,185 @@
+/*

```

Comment:
I'll need some time to look at this, but I can immediately say that this is missing a translator header. You should be using the file as saved in the Zotero translator folder, not something copy&pasted from Scaffold.